move credential format specific DCQL parameters to the annex (#340)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

`meta`: 
: OPTIONAL. An object defining additional properties requested by the Verifier that
apply to the metadata and validity data of the Credential. The properties of

> this object are defined per Credential Format in (#sd_jwt_vc_meta_parameter) and (#mdoc_meta_parameter). If omitted,

Reviewer: should we have a clause allowing for further profiles (outside of this spec)?

> this object are defined per Credential Format in (#sd_jwt_vc_meta_parameter) and (#mdoc_meta_parameter). If omitted,

Reviewer 2: ```suggestion
this object are defined per Credential Format in (#sd_jwt_vc_meta_parameter) and (#mdocs_meta_parameter). If omitted,
```

Author: ```suggestion
this object are defined per Credential Format. Examples of those are in (#sd_jwt_vc_meta_parameter) and (#mdocs_meta_parameter). If omitted,
```

be present more than once.

`path`:

> : REQUIRED if the Credential Format uses a JSON-based claims structure (e.g., IETF SD-JWT VC, W3C Verifiable Credentials and Anon Creds); MUST NOT

Reviewer 3: Isn't the AnonCreds data structure essentially a name/value list?

Author: ```suggestion
: REQUIRED if the Credential Format uses a JSON-based claims structure (e.g., IETF SD-JWT VC and W3C Verifiable Credentials); MUST NOT
```

be present otherwise. The value MUST be a non-empty array representing a claims path pointer that specifies the path to a claim
within the Verifiable Credential, as defined in (#claims_path_pointer).

`values`:
: OPTIONAL. An array of strings, integers or boolean values that specifies the expected values of the claim.
If the `values` property is present, the Wallet SHOULD return the claim only if the
type and value of the claim both match for at least one of the elements in the array. Details of the processing rules are defined in (#selecting_claims).

> The ISO mdoc specific parameters to be used in Claims Query are defined in (#mdocs_claims_query).

Reviewer: see above

### DCQL Query and Response

This section defines ISO mdoc specific DCQL Query and Response parameters.

> #### mdoc specific Parameters in the `meta` parameter in Credential Query {#mdocs_meta_parameter}

Reviewer: I think we could drop "mdoc specific Parameters in the" here as this whole section is about mdoc

> The following is an ISO mdoc specific parameter in the `meta` parameter in Credential Query defined in (#credential_query).

Reviewer 3: ```suggestion
The following is an ISO mdoc specific parameter in the `meta` parameter in a Credential Query as defined in (#credential_query).
```

`doctype_value`:
: OPTIONAL. String that specifies an allowed value for the
doctype of the requested Verifiable Credential. It MUST
be a valid doctype identifier as defined in [@ISO.18013-5].

> #### mdoc specific Parameters in Claims Query {#mdocs_claims_query}

Reviewer 3: ```suggestion
#### Parameters in the Claims Query {#mdocs_claims_query}
```

> The following are ISO mdoc specific parameters to be used in Claims Query defined in (#claims_query).

Reviewer 3: ```suggestion
The following are ISO mdoc specific parameters to be used in a Claims Query as defined in (#claims_query).
```

### DCQL Query and Response

This section defines SD-JWT VC specific DCQL Query and Response parameters.

> #### SD-JWT VC specific Parameters in the `meta` parameter in Credential Query {#sd_jwt_vc_meta_parameter}

Reviewer 3: ```suggestion
#### Parameters in the `meta` parameter in Credential Query {#sd_jwt_vc_meta_parameter}
```

> The following is an SD-JWT VC specific parameter in the `meta` parameter in Credential Query defined in (#credential_query).

Reviewer 3: ```suggestion
The following is an SD-JWT VC specific parameter in the `meta` parameter in a Credential Query as defined in (#credential_query).
```
